db/migrate/20150115155947_add_scaptimony_scap_content_digest.rb

```diff
@@ -0,0 +1,11 @@
+class AddScaptimonyScapContentDigest < ActiveRecord::Migration
+  def change
+    add_column :scaptimony_scap_contents, :digest, :string, :limit => 128
+    Scaptimony::ScapContent.find_each do |content|
```

Comment:
1) new installation may be fine becase there won't be any contents so no error will occure, but on some existing installations upgrade would fail

2) we should not update existing migration, if there's no blocker, I'd prefer to do it properly now (copy'n'paste should work and I'll be the first one who will test it even before merge if you want)